TLM/TLM/UI/IncompatibleModsPanel.cs

```diff
@@ -336,7 +336,11 @@ private void TryPopModal()
 
             if (blurEffect != null && UIView.ModalInputCount() == 0)
             {
-                ValueAnimator.Animate("ModalEffect", delegate (float val) { blurEffect.opacity = val; }, new AnimatedFloat(1f, 0f, 0.7f, EasingType.CubicEaseOut), delegate () { blurEffect.Hide(); });
+                ValueAnimator.Animate(
```

Comment:
It would be better if stuff like this went in separate PRs

Reply:
I think it is doable to separate all the whitespace refactorings and minor formatting and the actual changes for lane arrows. Some of them will go entirely into lane arrows PR. Then can we get the whitespace/formatting merged first?

Reply:
Yup, makes sense. Have to be careful the whitespace updates don't cause more merge conflicts with harmony branch though, that's already going to be nightmare to merge.

Reply:
https://github.com/krzychu124/Cities-Skylines-Traffic-Manager-President-Edition/pull/419

Reply:
I will merge master to harmony branch tomorrow ;)